pkg/apis/crd/v1beta1/types.go

```diff
@@ -229,6 +229,12 @@ type ExternalIPPoolSpec struct {
 	SubnetInfo *SubnetInfo `json:"subnetInfo,omitempty"`
 	// The Nodes that the external IPs can be assigned to. If empty, it means all Nodes.
 	NodeSelector metav1.LabelSelector `json:"nodeSelector"`
+
+	// FilterReservedIPs prevents allocation of IPs ending in .0 or .255.
+	// Only applicable to IP ranges defined with start and end. Ignored for CIDRs.
+	// +optional
+	// +kubebuilder:default=false
+	FilterReservedIPs bool `json:"filterReservedIPs,omitempty"`
```

Comment:
Maybe a general question, should we use `omitzero` (from go 1.24) for non-pointer field?